mempool/v0/clist_mempool_test.go

```diff
@@ -149,7 +149,7 @@ func TestReapMaxBytesMaxGas(t *testing.T) {
 		{20, 0, -1, 0},
 		{20, 0, 10, 0},
 		{20, 10, 10, 0},
-		{20, 24, 10, 1},
```

Comment:
I believe this has also been reverted (consistent with [this](https://github.com/celestiaorg/celestia-core/pull/898/files#r1046389376) being reverted)